pypesto/visualize/misc.py

```diff
@@ -1,7 +1,9 @@
+import copy
 import warnings
 from numbers import Number
-from typing import Iterable, List, Optional, Union
+from typing import Iterable, List, Optional, Sequence, Union
 
+import amici
```

Comment:
Tests are probably failing because of this. As much as I understand pyPESTO testing, for some tests, the virtual environment doesn't have amici installed. So if this file is imported, even if the functions using amici are not used, those tests will fail.
To fix this, I think the standard is to include a try: except: for the import, like this:
```
try:
    import amici
except ImportError:
    pass
```

In all hierarchical files, we do this for both amici and petab.